### DISCUSSION TITLE
A problem with linux

### DISCUSSION THREAD

**Discussion by Heshi99** [community/community#102592] [Programming Help] [answered]
### Body

Why does the process stop like this? what is the error

**The following signatures are invalid: EXPKEYSIG ED444FF07D8D0BF6 Kali Linux Repository devel@kali.org**
**" Some index files failed to download. They have been ignored, or old ones used instead. "**

Help solve this. Thank you

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by naveen-98** [comment, marked as answer]
The warning message "Some index files failed to download. They have been ignored, or old ones used instead" in Linux typically indicates an issue with the package repositories or the network connection.

Follow this for this:

Run the following command to update the package lists

`sudo apt-get update`

Then run this command :

`wget --no-check-certificate http.kali.org/kali/pool/main/k/kali-archive-keyring/kali-archive-keyring_2022.1_all.deb`

After this again issue `sudo apt update` command.

See this for more details =  [Linux](https://youtu.be/kH6HD2JQ0fY?si=bXy3OXhQR_BD9dcd)

**Reply by Heshi99** [reply]
It really works. thank you 🎇
